apps/air-discount-scheme/backend/src/environments/environment.ts

```diff
@@ -12,6 +12,18 @@ const devConfig = {
     url: process.env.NATIONAL_REGISTRY_URL,
     username: process.env.NATIONAL_REGISTRY_USERNAME,
     password: process.env.NATIONAL_REGISTRY_PASSWORD,
+    authMiddlewareOptions: {
+      forwardUserInfo: true,
```

Comment:
You should have forwardUserInfo set to false as it only adds the User-Agent X-Forwarded-For headers to the request, which Þjóðskrá doesn't need.